### PR TITLE
fix: add config for disabling direct upload verification

### PR DESCRIPTION
**Pull request by Michaelhobo**
Changelog: Title
Ticket: None

### CODE REVIEW

File: .gitpod.yml
Comment (oldgiova) [inline .gitpod.yml:1]: I'm not sure about this. Maybe is it better to add it into the `.gitignore`?

File: mender/values.yaml [hunk 1]
```diff
@@ -66,18 +66,19 @@ ingress:
   # alb.ingress.kubernetes.io/healthcheck-path: /ui/
   # cert-manager.io/issuer: "letsencrypt"
   path: /
-  extraPaths: []
+  extraPaths:
```
Comment (oldgiova) [inline mender/values.yaml:69]: Please add formatting to a different commit (or remove it entirely ;) )

**Review by oldgiova — CHANGES_REQUESTED** [review]
Thanks for your contribution! Can you please add the `.gitpod.yml` to the `.gitignore` and move formatting to another commit (or remove it)?

Thanks!

**Comment by tranchitella** [comment]
@Michaelhobo and @oldgiova, can you check if this PR can replace this one?
https://github.com/mendersoftware/mender-helm/pull/188

**Comment by oldgiova** [comment]
> @Michaelhobo and @oldgiova, can you check if this PR can replace this one?
> https://github.com/mendersoftware/mender-helm/pull/188

Yes I can confirm

File: .gitpod.yml
Comment (Michaelhobo) [reply .gitpod.yml:1]: Done

File: mender/values.yaml (hunk 1 above)
Comment (Michaelhobo) [reply mender/values.yaml]: Done.

**Comment by Michaelhobo** [comment]
Sorry for the delay, I was out sick last week.

**Comment by oldgiova** [comment]
> Sorry for the delay, I was out sick last week.

No, problem, thanks for the contribution! Although it looks good to me, probably the new PR https://github.com/mendersoftware/mender-helm/pull/188 is superseding this one, so I'm leaving the decision to @tranchitella .

**Comment by Michaelhobo** [comment]
Sounds good, either way works!

**Comment by tranchitella** [comment]
@Michaelhobo and @oldgiova, I merged https://github.com/mendersoftware/mender-helm/pull/188

**Pull request closed without merging**